### PR TITLE
Update EOL for Alpine Linux release

### PR DESCRIPTION
**Pull request by kolenichsj**
Added EOL date for Alpine Linux 3.14

### CODE REVIEW

**Comment by kolenichsj** [comment]
Added EOL date for Alpine Linux 3.15

File: db/software-eol.db [hunk 1]
```diff
@@ -16,6 +16,8 @@
 #
 # Alpine - https://alpinelinux.org/releases/
 #
+os:Alpine 3.15:2023-11-01:1169879680
```
Comment (Jaimie85) [inline db/software-eol.db:19]: There seems to be an error in the converted time

**Review by Jaimie85 — CHANGES_REQUESTED** [review]
There seems to be an error in the converted time.

File: db/software-eol.db (hunk 1 above)
Comment (mboelen) [reply db/software-eol.db]: Fixed with my change

**Comment by kolenichsj** [comment]
Thank you for the correction

**Pull request merged**